BoxContentSDK/BoxContentSDK/Requests/BOXFolderItemsRequest.h

```diff
@@ -12,6 +12,7 @@
 
 @property (nonatomic, readwrite, strong) NSString *folderID;
 @property (nonatomic, readwrite, assign) BOOL requestAllItemFields;
+@property (nonatomic, readwrite, assign) BOOL requestPerformantItemFields;
```

Comment:
Where does it make sense to add a comment that explains this? Unfortunately 'performant item fields' is not as self explanatory or clear as 'all item fields' IMO.